shared/src/arguments.rs

```diff
@@ -120,3 +120,13 @@ pub fn wei_from_gwei(s: &str) -> anyhow::Result<f64> {
     let in_gwei: f64 = s.parse()?;
     Ok(in_gwei * 10e9)
 }
+
+pub fn default_amount_to_estimate_prices_with(network_id: &str) -> Option<U256> {
+    match network_id {
+        // Mainnet, Rinkeby
+        "1" | "4" => Some(10u128.pow(18).into()),
+        // Xdai
+        "100" => Some(10u128.pow(21).into()),
```

Comment:
These amounts seem weird, you have 1*10^18 and 100*10^21 I think the 100 already make up for the +2 in the exponent.

Reply:
It shouldn't be multiplication unless I'm missing something. The 100 is just matching on the network id which is `100` for xdai. The reason for this amount is that 10e21 should be ~ 1000 usd on xdai.

Reply:
Oh ha ha! Silly me. I just read it wrong.